cmd/statusd/main.go

```diff
@@ -253,7 +253,7 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		nodeConfig.ClusterConfig.BootNodes = nil
 	}
 
-	nodeConfig.Discovery = *discovery
```

Comment:
I'd keep `Discovery()` as a method, that inverts the `NoDiscovery` flag, so we don't have to have double inverse logic ("not no disovery") in many places.

Reply:
isn't it actually 1 place?

Reply:
ah, yeah, though it still looks weird :)
anyway, not critical at all